tests/test_default.py

```diff
@@ -0,0 +1,12 @@
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    '.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_hosts_file(File):
```

Comment:
Given how [trivial this role is (installs a single package, should we consider deprecating it?)](https://github.com/joshmoore/ansible-role-analysis-tools/blob/ffd5e16c641f7347c160a59b33d0f73e977136af/tasks/main.yml) you might as well make this a proper test such as:
```
def test_analysis_packages(Package):
    assert Package('parallel')
```